roles/ceph-common/templates/redhat_storage_repo.j2

```diff
@@ -1,22 +1,22 @@
 # {{ ansible_managed }}
 [rh_storage_mon]
-name=Red Hat Ceph Storage - local packages for Ceph
+name=Red Hat Ceph Storage - local packages for Ceph monitor
 baseurl=file://{{ ceph_stable_rh_storage_repository_path }}/MON
 enabled=1
 gpgcheck=1
 type=rpm-md
 priority=1
 
 [rh_storage_osd]
-name=Red Hat Ceph Storage - local packages for Ceph
+name=Red Hat Ceph Storage - local packages for Ceph OSD
 baseurl=file://{{ ceph_stable_rh_storage_repository_path }}/OSD
 enabled=1
 gpgcheck=1
 type=rpm-md
 priority=1
 
 [rh_storage_tools]
-name=Red Hat Ceph Storage - assorted tools
+name=Red Hat Ceph Storage - local packages for Ceph client, MDS, and RGW
```

Comment:
rbd-mirror is in there, you're right. It was getting a little long so I didn't list everything :)